com.stansassets.scene-management/BuildConfigurator/Editor/EditorBuildSettingsValidator.cs

```diff
@@ -6,14 +7,27 @@ namespace StansAssets.SceneManagement.Build
     [InitializeOnLoad]
     public class EditorBuildSettingsValidator
     {
-        public const string ScenesSyncDescription = "Current Editor Build Settings are our of sync " +
-                                                    "with the Scene Management build configuration.";
+        public const string ScenesSyncWarningDescription = "Current Editor Build Settings are our of sync " +
+                                                           "with the Scene Management build configuration.";
 
-        const string k_HintDescription= "Scenes can be synchronized through the " +
-                                        "'Scene Management -> Build Settings'.";
+        public const string ScenesSyncOkDescription = "Editor Build Settings are synced " +
+                                                      "with the Scene Management build configuration.";
 
         public static readonly Color OutOfSyncColor = new Color(0.93f, 0.39f, 0.32f);
+        
+        public const string ScenesDuplicatesWarningDescription = "Repetitive scenes found. Resolve it manually!";
 
+        public const string ScenesDuplicatesOkDescription = "There are no repetitive scenes.";
+
+        public static readonly Color DuplicateColor = new Color(1f, 0.78f, 1f);
+
+        const string k_HintDescription = "Scenes can be synchronized through the " +
```

Comment:
All public endpoints have to be documented.
Why do those fields have to be public? How do you anticipate developers using it?